### PR TITLE
Poker task example

### PR DESCRIPTION
**Pull request by MaximKarpenko**

### CODE REVIEW

File: src/ua/hillel/lesson5/Poker.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+import java.util.Random;
+
+/**
+ * @author Maxim Karpenko mkarpenko@modeln.com
+ */
+
+public class Poker {
+  private static final int NUMBER_OF_CARDS = 52;
+  private final static int NUM_OF_PLAYERS = 4;
```
Comment (apollyonix) [inline src/ua/hillel/lesson5/Poker.java]: change final and static

File: src/ua/hillel/lesson5/Poker.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+  private static void serveCards() {
+    int index = 0;
+    for (int round=1; round<=5; round++) {
+      System.out.println("Round: " + round);
+
+      for (int player=1; player<=NUM_OF_PLAYERS; player++) {
+        System.out.println("Player " + player + " gets: " + deck[index++]);
+      }
+
```
Comment (apollyonix) [inline src/ua/hillel/lesson5/Poker.java]: remote extra line

**Review by apollyonix — CHANGES_REQUESTED** [review]

File: src/ua/hillel/lesson5/Poker.java (hunk 1 above)
Comment (MaximKarpenko) [reply src/ua/hillel/lesson5/Poker.java]: replaced

File: src/ua/hillel/lesson5/Poker.java (hunk 2 above)
Comment (MaximKarpenko) [reply src/ua/hillel/lesson5/Poker.java]: deleted

**Review by apollyonix — APPROVED** [review]

**Pull request merged**